src/main/java/se/sundsvall/seabloader/integration/invoicecache/InvoiceCacheClient.java

```diff
@@ -18,6 +19,6 @@ public interface InvoiceCacheClient {
 	 * 
 	 * @param invoicePdfRequest with attributes for export an invoice.
 	 */
-	@PostMapping(path = "importInvoice", produces = APPLICATION_PROBLEM_JSON_VALUE)
+	@PostMapping(path = "invoices", consumes = APPLICATION_JSON_VALUE, produces = APPLICATION_PROBLEM_JSON_VALUE)
 	ResponseEntity<Void> importInvoice(@RequestBody InvoicePdfRequest invoicePdfRequest);
```

Comment:
I think that the method name in the client should be constructed from the view-point of this service.

(This service doesn't import anything. It sends/stores/trasmits invoices.)